gitbook-docs/security.md

```diff
@@ -89,27 +89,23 @@ If the login fails, the server will send a response like the following:
   "result": 401
 }
 ```
-The `result` codes are the same as normally used in HTTP
+The `result` codes are the same as normally used in HTTP.
 
 ### Providing Authorization to the Server in Subsequent Requests
 
 #### Web Based Clients
 
-Web based clients should be sure to include the cookie set in the authentication response in all subsequent requests.
-
-```
-Cookie: SK_TOKEN=eyJhbGciOiJIUzUxMiJ9.eyJyb2xlcyI6IltcInNraXBwZXJcIl0iLCJpYXQiOjE1ND...
-
-```
+Web based clients should be sure to include the cookie or `Authorization` HTTP header obtained from the authentication response in all subsequent requests.
 
 #### WebSockets Clients
 
-Clients can include the authentication cookie with the initial request. 
+Clients can include the authentication cookie with the initial request.
 
 Clients can include the `Authorization` HTTP header with the initial connect request. The format of the header should
-be `{type} {token}`, for example `Authorization: JWT eyJhbGciOiJIUzI1NiIsI...ibtv41fOnJObT4RdOyZ_UI9is8`
+be `BEARER {token}`, for example `Authorization: BEARER eyJhbGciOiJIUzI1NiIsI...ibtv41fOnJObT4RdOyZ_UI9is8`
```

Comment:
Only the first letter of `Bearer` should be capitalized.

Reply:
Also, this is a breaking change wrt. current implementation but I think the current implementation is non-conforming, so I don't have an issue with that.

Reply:
Changed